app/src/components/homePageMainContent/homePageMainContent.tsx

```diff
@@ -1,32 +1,53 @@
 import React from "react";
 import FeaturedContentComponent from "../featuredContent/featuredContent";
 import { useEffect, useState } from "react";
-import CollectionLanes from "../lanes/collectionLanes/collectionLanes";
-import CollectionLanesLoading from "../lanes/collectionLanes/collectionLanesLoading";
 import { DC_URL } from "@/src/config/constants";
+import { Lane as DCLane } from "../lane/lane";
+import LaneDataType from "@/src/types/Lane";
+import LaneLoading from "../lane/laneLoading";
 
-const HomePageMainContent = ({ data }) => {
+interface HomePageMainContentProps {
+  swimlanes: LaneDataType[];
+  randomNumber: number;
+}
+
+const HomePageMainContent = ({
+  swimlanes,
+  randomNumber,
+}: HomePageMainContentProps) => {
   const [isLoaded, setIsLoaded] = useState(false);
 
   useEffect(() => {
     setIsLoaded(true);
   }, []);
 
+  const firstSwimLane = swimlanes[0];
   return isLoaded ? (
     <>
-      <CollectionLanes
-        lanesWithNumItems={[data.lanesWithNumItems[0]]}
-        seeMoreLink={`${DC_URL}/collections/lane`}
-      />
-      <FeaturedContentComponent randomNumber={data.randomNumber} />
-      <CollectionLanes
-        lanesWithNumItems={data.lanesWithNumItems.slice(1)}
+      <div style={{ marginTop: "-3rem" }} />
+      <DCLane
+        key={0}
```

Comment:
I think this will conflict with the `0` below in the map. Do you see anything in the console?

Reply:
No console issues (at least not more than we usually have)